api/envoy/config/filter/accesslog/v2/accesslog.proto

```diff
@@ -188,3 +191,30 @@ message ResponseFlagFilter {
     ]
   }];
 }
+
+// Filters requests based on the gRPC status. If a gRPC status is not provided, it will infer the
+// status from the HTTP code.
+message GrpcStatusFilter {
+  // Logs only responses that have any one of the gRPC statuses in this field.
```

Comment:
I was thinking about a similar problem while I was originally writing it. It would be really cumbersome to write out all 15 other codes if the user is only interested in non-`OK`, and not that hard to add a boolean, it really just entails:

1. Changing the proto
2. Changing the definition in `access_log_impl.h` to store that `bool` in the class.
3. Changing the return in `GrpcStatusFilter::evaluate` to be reversed if the bool is active.

Reply:
Feature added in 93eb881.